### PR TITLE
Vadym biletskyi

### PR DESCRIPTION
**Pull request by umpire-3**
First scope

### CODE REVIEW

File: Vadym_Bilestkyi/1/Task8.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def is_valid_time(time):
+    try:
+        hours, minutes = time.split(':')
```
Comment (oleksandr-medviediev) [inline Vadym_Bilestkyi/1/Task8.py]: Wrong exception handling usage. Avoid it for now.

File: Vadym_Bilestkyi/1/Task8.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    except ValueError:
+        return False
+
+    return 0 <= int(hours) < 24 and 0 <= int(minutes) < 60
```
Comment (oleksandr-medviediev) [inline Vadym_Bilestkyi/1/Task8.py]: Don't return expressions!

File: Vadym_Bilestkyi/2/Task3.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+def custom_map(func, iterable):
+    return (func(element) for element in iterable)
```
Comment (oleksandr-medviediev) [inline Vadym_Bilestkyi/2/Task3.py]: Don't return expressions.

File: Vadym_Bilestkyi/2/Task1.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+def find_odd1(numbers):
+    for number1 in numbers:
+        count = 0
+        for number2 in numbers:
+            if number1 == number2:
+                count += 1
+        if count % 2 == 1:
+            return number1
+
+    return None
```
[the remark this answers is not in the document]
Comment (umpire-3) [reply Vadym_Bilestkyi/2/Task1.py]: I know, but you said that we can return None explicitly if we want.

File: Vadym_Bilestkyi/2/Task2.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+def array_diff1(array1, array2):
+    result = array1[:]
+    for element in array2:
+        while True:
+            try:
+                result.remove(element)
```
[the remark this answers is not in the document]
Comment (umpire-3) [reply Vadym_Bilestkyi/2/Task2.py]: why? Is there any other way to remove in loop all copies of same element without exception handling.

File: Vadym_Bilestkyi/2/Task1.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (oleksandr-medviediev) [reply Vadym_Bilestkyi/2/Task1.py]: Yeah, that's why I wrote "You can" instead "You have to".

File: Vadym_Bilestkyi/2/Task2.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (oleksandr-medviediev) [reply Vadym_Bilestkyi/2/Task2.py]: You can use list comprehension.

**Pull request merged**